<?xml version="1.0" ?>
<launch>

	<rosparam command="load" file="$(find elisa3_node_cpp)/launch/params.yaml" />

    <node pkg="elisa3_node_cpp" type="elisa3" name="elisa3_1" output="screen"/>

    <param name="robot_description" textfile="$(find elisa3_node_cpp)/urdf/elisa3_urdf.xml"/>
    <node name="elisa3_state_publisher" pkg="robot_state_publisher" type="state_publisher">
        <param name="tf_prefix" value="$(arg elisa3_name)" />
    </node>
    <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find elisa3_node_cpp)/config/elisa3_single_rviz.rviz"/>

</launch>
